pycbc/population/live_pastro_utils.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from . import live_pastro as livepa
+
+
+def insert_live_pastro_option_group(parser):
+    """ Add low-latency p astro options to the argparser object.
+
+    Parameters
+    ----------
+    parser : object
+        ArgumentParser instance.
+
+    Returns
+    -------
+    live_pastro_group :
+        Argument group object
+    """
+
+    live_pastro_group = parser.add_argument_group('Options for live p_astro '
+                                                  'calculation')
+
+    # Only one choice so far, allow for more later
+    live_pastro_group.add_argument('--p-astro-method',
+                                   choices=['template_param_bins'])
+    live_pastro_group.add_argument('--p-astro-spec-data',
+        help='File containing information to set up p_astro calculation')
+
+    return live_pastro_group
+
+
+def verify_live_pastro_options(args):
+    """ Input check for live p_astro options
+
+    Parameters
+    ----------
+    args : argparse Namespace object
+        Populated namespace output by parse_args()
+
+    Returns
+    -------
+    args : argparse Namespace object
+    """
+    if args.p_astro_method is not None and args.p_astro_spec_data is None:
+        raise RuntimeError('Need a p astro data spec file for method %s! ' %
+                           args.p_astro_method)
+    if args.p_astro_method is None and args.p_astro_spec_data is not None:
+        raise RuntimeError('Need to specify p_astro method to do a p_astro '
```

Comment:
If we do want to keep this code as is, these errors should not be exceptions (and certainly not RuntimeErrors), but calls to `parser.error()`. So you would need to pass the parser object as an extra arg to this function (yes, I know we do not do that in other `verify_*` calls, but we should!).

Reply:
Simplified option structure means these all go away ..